fontforge_font_creator/creator.py

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env python3
+import fontforge
+import os
+import ruamel.yaml
+import shutil
+
+
+class FontMaker:
+    """
+    Usage Example:
+        fm = FontMaker('osmaxx_v1_definition.yml')
+        fm.create_font()
+    """
+    def __init__(self, font_config):
+        self.config = self.yml_reader(font_config)
+        self._base_path = '/home/fonts/osmaxx/svg'
+        self.n = fontforge.font()
+        self.n.encoding = 'Unicode'
+
+    def yml_reader(self, font_config):
```

Comment:
I guess this function returns the read content, not an object capable of reading (a "reader"). So a better name might be `read_yaml`. Also, this function doesn't care what YAML file it reads, so the argument should probably be named `filename` or similar.